src/components/Header.js

```diff
@@ -18,36 +18,26 @@
 // Created by Jason Leach on 2018-09-04.
 //
 
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
+import React from 'react';
+import PropTypes from 'prop-types';
 import AuthButton from './AuthButton';
 import logo from './bcgovlogo.svg';
 import './Header.css';
 
-class Header extends Component {
-  render() {
-    return (
-      <header>
-        <img src={logo} className="header-logo" alt="logo" />
-        <p className="header-title">Secure Sign</p>
-        <div className="right-push">
-          <AuthButton />
-        </div>
-      </header>
-    );
-  }
-}
+const Header = ({ authentication }) => {
+  return (
+    <header>
+      <img src={logo} className="header-logo" alt="logo" />
+      <p className="header-title">Secure Sign</p>
```

Comment:
This could be set as a header tag instead of a paragraph for semantics, not critical as i don't think browsers could crawl these pages anyhow as they are built on run time